olm/open-liberty-crd.yaml

```diff
@@ -363,8 +363,10 @@ spec:
                   properties:
                     cpu:
                       type: string
+                      pattern: '^[0-9]+m?$'
```

Comment:
This line does appear to handle fractions (i.e. 0.1 = 100m).
Doc: ""The upper limit of CPU core. Specify integers, fractions (e.g. 0.5), or millicores values(e.g. 100m, where 100m is equivalent to .1 core).""

Reply:
OK...will update to support fractions.